reloc.ml

```diff
@@ -251,7 +251,16 @@ let quote_files cmdline lst =
 let cygpath l cont =
   let accept_error = (!use_cygpath = `Try && l <> []) in
   let l =
-    let args = String.concat " " (List.map Filename.quote l) in
+    let s = String.concat " " (List.map Filename.quote l) in
+    let args =
+      if String.length s >= 1024 then begin
```

Comment:
I suggest factoring this constant with the one in `quote_files`. Something like:

```ocaml
(* Use response files if the command line length exceeds 1024
   characters (friendlier for humans than 8192). *)
let max_command_line_length = 1024
```

Reply:
My bad, there's already this:

```ocaml
(* This is the longest command which can be passed to [Sys.command] *)
let max_command_length =
  let processor = try Sys.getenv "COMSPEC" with Not_found -> "cmd.exe" in
  (* The 4 is from the " /c " *)
  8191 - String.length processor - 4
```

Let's keep 1024.